core-bundle/src/Resources/contao/dca/tl_content.php

```diff
@@ -1074,6 +1080,87 @@ public function preserveReferenced()
 		}
 	}
 
+	/**
+	 * Filter the content elements
+	 */
+	public function filterContentElements()
+	{
+		if ($this->User->isAdmin)
+		{
+			return;
+		}
+
+		if (empty($this->User->elements))
+		{
+			$GLOBALS['TL_DCA']['tl_content']['config']['closed'] = true;
+		}
+		elseif (!\in_array($GLOBALS['TL_DCA']['tl_content']['fields']['type']['default'], $this->User->elements))
```

Comment:
I just stubeled upon this in `ce-access`: since we have migrated the default values from DCA to the SQL definition, the default type will always be empty.

Reply:
You are right. Fixed in 8a193acb0db94898c8622f0842eb3ee53e23c23e.